configure.ac

```diff
@@ -1640,30 +1640,33 @@ AC_SUBST(LDFLAGS_NODIST)
 UNIVERSAL_ARCH_FLAGS=
 AC_SUBST(UNIVERSAL_ARCH_FLAGS)
 
+dnl PY_CHECK_CC_WARNING(ENABLE, WARNING, [MSG])
+AC_DEFUN([PY_CHECK_CC_WARNING], [
+  AC_MSG_CHECKING(m4_ifblank([$3], [if we can $1 $CC $2 warning], [$3]))
+  AS_VAR_PUSHDEF([py_var], [ac_cv_$1_]m4_normalize($2)[_warning])
+  AS_VAR_COPY([py_cc], [CC])
+  AS_VAR_SET([CC], ["$CC -W$2 -Werror"])
```

Comment:
Is ``CC`` the correct variable or should you rather override ``CFLAGS`` or ``BASECFLAGS``?

Reply:
The old code did override `CC`, but you are correct, it is cleaner to use `CFLAGS`. I'll fix it.